### PR TITLE
Fixed warm-reboot for BFN platform

### PR DESCRIPTION
**Pull request by akokhan**
Signed-off-by: Andriy Kokhan <akokhan@barefootnetworks.com>

**- How to verify it**
Run warm-reboot CTs

### CODE REVIEW

**Comment by akokhan** [comment]
This change is needed for 201911 as well

**Comment by akokhan** [comment]
retest default please

**Comment by akokhan** [comment]
retest this please

**Comment by akokhan** [comment]
retest default please

**Comment by akokhan** [comment]
retest this please

File: scripts/fast-reboot [hunk 1]
```diff
@@ -316,9 +316,6 @@ case "$REBOOT_TYPE" in
             # source mlnx-ffb.sh file with
             # functions to check ISSU upgrade possibility
             source mlnx-ffb.sh
-        elif [[ "$sonic_asic_type" == "barefoot" ]]; then
-            REBOOT_TYPE="fastfast-reboot"
-            BOOT_TYPE_ARG="fastfast"
```
Comment (lguohan) [inline scripts/fast-reboot:321]: the title does not match the code. the code is in the warm-reboot path. why the title says fix fast-reboot for bfn platform?

Comment (lguohan) [reply scripts/fast-reboot:321]: meanwhile, if the intention is the fix the warm reboot path, then that is different from my understand how warm reboot works on barefoot path. My understand of warm reboot on barefoot platform is that you do fast re-init the asic. 

What is the difference between the fast reboot and warm reboot on your platform?

Comment (akokhan) [reply scripts/fast-reboot:321]: @lguohan , thank you for your comments. You are right, its warm-reboot changes. Fixed PR title. The warm-reboot approach on BFN platform has been changed in the latest SDK package https://github.com/Azure/sonic-buildimage/pull/4397 in both master and 201911.

**Comment by akokhan** [comment]
retest this please

**Comment by akokhan** [comment]
@lguohan could you merge this please? Thanks

**Comment by akokhan** [comment]
Is there anything that blocks this from being merged?

**Comment by akokhan** [comment]
@jleveque , @lguohan , could you please merge this PR? This affects warm-boot testing on BFN platform. Thanks.

**Comment by akokhan** [comment]
@lguohan , @jleveque , could you please approve and merge this?

**Comment by akokhan** [comment]
@lguohan , @prsunny , please approve and merge.

**Comment by akokhan** [comment]
@lguohan could you please merge this? Thank you.

**Pull request merged**